eng/pipelines/mgmt-ci.yml

```diff
@@ -11,7 +11,6 @@ trigger:
       - sdk/keyvault/keyvault-certificates/
       - sdk/keyvault/keyvault-keys/
       - sdk/keyvault/keyvault-secrets/
-      - sdk/keyvault/keyvault/
```

Comment:
Why was this removed?

Reply:
There is no such path hat currently exists in the repo